### PR TITLE
{lang}[gfbf/2023a] R v4.3.2

### PR DESCRIPTION
**Pull request by SebastianAchilles**
Alternative for https://github.com/easybuilders/easybuild-easyconfigs/pull/18221

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, Intel(R) Core(TM) i7-10700 CPU @ 2.90GHz (skylake), 1 x NVIDIA NVIDIA RTX A4000, 545.23.06, Python 3.6.8
See https://gist.github.com/SebastianAchilles/e1974545cd1a8b6f689de6f8da073767 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/949b93a7176ae17fd39b2a9a16a0f6bb for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.16
See https://gist.github.com/SebastianAchilles/f5c734c7704107286e6994073140a04b for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16
EB_ARGS="--experimental --parallel-extensions-install"

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso
CORE_CNT=16
EB_ARGS="--experimental --parallel-extensions-install"

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 4 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/0c35c4b32d76e97b961e1b6c92fc21f4 for a full test report.

**Comment by verdurin** [comment]
Hmm, problem building the dependency `libGLU`:
```
== 2023-11-10 10:42:51,013 run.py:246 INFO running cmd:  ./configure --prefix=/eb/maint/software/libGLU/9.0.3-GCCcore-12.3.0
== 2023-11-10 10:42:51,071 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:126 in __init__): cmd " ./configure --prefix=/eb/maint/software/libGLU/9.0.3-GCCcore-12.3.0 " exited with exit code 127 and output:
/bin/bash: ./configure: No such file or directory
 (at easybuild/tools/run.py:681 in parse_cmd_output)
 ```

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-centos-79 - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/SebastianAchilles/72871fa66635f41b0b94565e1843f675 for a full test report.

**Comment by SebastianAchilles** [comment]
> Hmm, problem building the dependency `libGLU`:
> 
> ```
> == 2023-11-10 10:42:51,013 run.py:246 INFO running cmd:  ./configure --prefix=/eb/maint/software/libGLU/9.0.3-GCCcore-12.3.0
> == 2023-11-10 10:42:51,071 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:126 in __init__): cmd " ./configure --prefix=/eb/maint/software/libGLU/9.0.3-GCCcore-12.3.0 " exited with exit code 127 and output:
> /bin/bash: ./configure: No such file or directory
>  (at easybuild/tools/run.py:681 in parse_cmd_output)
> ```

I found another problem with `libGLU` this morning: https://github.com/easybuilders/easybuild-easyconfigs/pull/19186
But your problem looks different.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node044.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/68b1e1c0a85ab96c2dba7885405f09cc for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node043.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/2f07663bcf1d0d7721e8a6d66d55a278 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node048.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/ced050c45c9ca4ee3d1fd242c7495447 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node042.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/b1ff8f6597dca5a566f359acbed72643 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node049.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/d31d834bb3add87ba759770e3a32ce9f for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node043.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/a5c9ff8c0167bc5854106064c624db22 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node044.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/6e30a204877e5655a449debc2049c4e2 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node041.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/b2480dd6a0cd2ca74361a72e2b91b7ae for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node048.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/05d82c8ec75ed30f5d030f525b2f74c4 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node045.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/981b2dbe05dbf7e7534e44a1b38ac5fe for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node041.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/304014c5ec0b9f3104c3cf74d837593c for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node044.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/72ce89c6ca6bf205b5121114b94ee5bb for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node043.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/e293e11f29a4ab42c61e0dc12736d78f for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node042.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/f8a5cca77179b683b9bbe32f5a693530 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node048.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/803bf2ac23b44c8a28328eab1336ea10 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node044.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/08e40a1460ce0752fcb903c1ad9766e1 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node049.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/bc578f0bba640527742ce6528123e086 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node042.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/20dfafe09c94c02d1e62086716763b06 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node046.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/1ffd877620b50f49d5c6a221dc325940 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node047.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/c32d55abb909809b12e499f42041033b for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node043.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/461ee3f591e055942b1b55a0a27e7e56 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node042.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/258198d644afcc22031941c56725f99d for a full test report.

**Comment by jfgrimm** [comment]
(sorry for the spam)

**Review by casparvl — COMMENTED** [review]
One question on the removal of `Xvfb` and `Cmake`. Otherwise, this PR looks good to me. I don't think it should hang on the failing build of @verdurin since that failed in a dependency (and is thus not directly related to the change in _this_ PR). Especially considering we have a fair amount of succesful test reports.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node200.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz (broadwell), Python 3.6.8
See https://gist.github.com/lexming/a2f9be62c185705533d9ba489d93d853 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/lexming/34872fde418d3a79a8c8c91abdc704c3 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @verdurin and @SebastianAchilles for the hard work !

File: easybuild/easyconfigs/r/R/R-4.3.2-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['b3f5760ac2eee8026a3f0eefcb25b47723d978038eee8e844762094c860c452a']
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+    ('Autotools', '20220317'),
```
[the remark this answers is not in the document]
Comment (SebastianAchilles) [reply easybuild/easyconfigs/r/R/R-4.3.2-gfbf-2023a.eb:16]: This R easyconfig is lightweight, as it only contains `R` itself, the `devtools` extension and the required dependencies for it. This is why `Xvfb` and `CMake` are no longer needed in this easyconfig.

All other extensions are in `R-bundle-CRAN` https://github.com/easybuilders/easybuild-easyconfigs/pull/19170 which also contains `Xvfb` and `CMake` as `builddependencies`.

[the remark this answers is not in the document]
Comment (casparvl) [reply easybuild/easyconfigs/r/R/R-4.3.2-gfbf-2023a.eb:16]: Clear, thanks! :)

**Pull request merged**